packages/core-plugin-api/api-report.md

```diff
@@ -756,4 +759,8 @@ export function withApis<T>(apis: TypesToApiRefs<T>): <P extends T>(
   (props: React_2.PropsWithChildren<Omit<P, keyof T>>): JSX.Element;
   displayName: string;
 };
+
+// Warnings were encountered during analysis:
+//
+// /backstage/dist-types/packages/core-app-api/src/app/types.d.ts:277:5 - (ae-forgotten-export) The symbol "AppIcons" needs to be exported by the entry point index.d.ts
```

Comment:
I'm having some issues fixing this and could use some help. I'm also not sure why this updated in `core-plugin-api` when the changes are in `core-app-api`, slightly confused

Reply:
I think it's because they are rexported from the `app-api` in `plugin-api`. 

Basically this is saying that you'll also need to export `AppIcons` out too.

Reply:
That was it. I got myself mixed up between the two as they have several items that look like the exact same. hopefully the build will go green now